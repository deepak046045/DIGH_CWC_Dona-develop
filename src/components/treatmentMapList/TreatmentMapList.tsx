'use client';

import React from 'react';
import layout from '@/scss/themes/layout.module.scss';

export default function TreatmentMapList() {
  return (
    <div
      className={`TreatmentMapList ${layout.gutter}`}
      style={{
        borderTop: '1px solid black',
        borderBottom: '1px solid black',
        background: 'lightgrey',
        color: 'white',
        display: 'flex',
        alignContent: 'center',
        justifyContent: 'center',
        padding: '20px 0',
      }}
    >
      <p style={{ textAlign: 'center', margin: '0' }}>
        Treatment Map List Placeholder
      </p>
    </div>
  );
}
