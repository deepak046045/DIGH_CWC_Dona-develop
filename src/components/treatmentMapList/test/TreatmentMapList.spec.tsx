import React from 'react';
import { render } from '@testing-library/react';
import TreatmentMapList from '../TreatmentMapList';

describe('TreatmentMapList', () => {
  it('should render page', () => {
    const { container } = render(<TreatmentMapList />);

    expect(container).toMatchSnapshot();
  });
});
