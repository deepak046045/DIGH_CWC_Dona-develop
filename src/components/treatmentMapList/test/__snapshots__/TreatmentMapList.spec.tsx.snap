// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TreatmentMapList should render page 1`] = `
<div>
  <div
    class="TreatmentMapList gutter"
    style="border-top: 1px solid black; border-bottom: 1px solid black; background: lightgrey; color: white; display: flex; align-content: center; justify-content: center; padding: 20px 0px;"
  >
    <p
      style="text-align: center; margin: 0px;"
    >
      Treatment Map List Placeholder
    </p>
  </div>
</div>
`;
